apps/meteor/app/slackbridge/server/slackbridge.js

```diff
@@ -30,14 +30,21 @@ class SlackBridgeClass {
 			this.rocket.clearSlackAdapters();
 
 			const tokenList = this.apiTokens.split('\n');
-			tokenList.forEach((apiToken) => {
-				const slack = new SlackAdapter(this);
-				slack.setRocket(this.rocket);
-				this.rocket.addSlack(slack);
-				this.slackAdapters.push(slack);
 
-				slack.connect(apiToken);
-			});
+			try {
+				tokenList.forEach((apiToken) => {
+					const slack = new SlackAdapter(this);
+					slack.setRocket(this.rocket);
+					this.rocket.addSlack(slack);
+					this.slackAdapters.push(slack);
+
+					slack.connect(apiToken);
+				});
+			} catch (err) {
+				connLogger.error('error connecting to slack', err);
+
+				throw err;
+			}
```

Comment:
changing the scope of your try/catch, your application becomes more "sensitive", before, it would try all tokens, even if some failed, now it stops at the first token that fails

Reply:
removed try/catch